### PR TITLE
Add interactive transcript

### PR DESCRIPTION
**Pull request by eoinsha**

### CODE REVIEW

File: src/_includes/episodes_layout.njk [hunk 1]
```diff
@@ -22,8 +22,12 @@
           Published {{ publish_date | formatDate }} -
           <a href="{{ spotify_link }}" class="text-primary-100 dark:text-secondary-100 hover:underline">Listen on your favourite podcast player</a>
         </p>
+        <div class="aspect-w-16 aspect-h-9">
+          <iframe🤽‍♂️ id="youtubePlayer" width="100%" height="100%"
```
Comment (lmammino) [inline src/_includes/episodes_layout.njk:26]: why is there a water polo emoji here? 😱

File: src/_includes/episodes_layout.njk [hunk 2]
```diff
@@ -22,8 +22,12 @@
...
+            src="https://www.youtube.com/embed/{{ youtube_id }}?enablejsapi=1" title="YouTube video player" frameborder="0" 
+            allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+        </div>
       </div>
-      <p>{{ youtube_id | youtubeLink }}</p>
```
Comment (lmammino) [inline src/_includes/episodes_layout.njk:26]: we can probably remove the `youtubeLink` helper at this stage then... I don't think we are using it somewhere else.

File: src/_includes/episodes_layout.njk [hunk 3]
```diff
@@ -22,8 +22,12 @@
           Published {{ publish_date | formatDate }} -
           <a href="{{ spotify_link }}" class="text-primary-100 dark:text-secondary-100 hover:underline">Listen on your favourite podcast player</a>
         </p>
+        <div class="aspect-w-16 aspect-h-9">
```
Comment (lmammino) [inline src/_includes/episodes_layout.njk:25]: do we need the tailwind aspect plugin at all?

can't we just use `aspect-video` or `aspect-[16/9]`?

https://tailwindcss.com/docs/aspect-ratio#setting-the-aspect-ratio

**Review by lmammino — APPROVED** [review]
Added some small comments. Awesome feature! Thanks :)

File: src/_includes/episodes_layout.njk (hunk 1 above)
Comment (eoinsha) [reply src/_includes/episodes_layout.njk:26]: If you can guess, I'll be very impressed. Hint: It's something Vim-related 😝

File: src/_includes/episodes_layout.njk (hunk 2 above)
Comment (eoinsha) [reply src/_includes/episodes_layout.njk:26]: It's still used for YouTube comments link and on `index.njk`. May be removable in the future.

**Pull request merged**